### DISCUSSION TITLE
Auto bed mesh - strange behavior

### DISCUSSION THREAD

**Discussion by alexandruvoicu** [mriscoc/Ender3V2S1#1047] [Q&A]
Hi, does anyone know how to setup bed mesh to check entire surface of the bed not only a square in the middle. Margins are skipped. I'm using a matrix of 9x9 points which is condensed only in center of the bed. Here is an picture that shows how auto bed mesh is working.
![Bed mesh](https://github.com/mriscoc/Ender3V2S1/assets/57235677/203acae2-ba67-4350-8ba5-464a7a43ffe5)

**Comment by mriscoc** [comment]
Check https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
